addons/account/models/account_move.py

```diff
@@ -2394,8 +2403,8 @@ def _get_computed_business_vals(self, price_unit, quantity, discount, currency,
             res['price_total'] = res['price_subtotal'] = subtotal
         return res
 
-    @api.model
-    def _get_computed_accounting_vals(self, price_subtotal, move_type, currency, company, date):
+    @api.multi
+    def _get_computed_accounting_vals(self, price_subtotal=None, move_type=None, currency=None, company=None, date=None):
```

Comment:
rename into _get_fields_onchange_subtotal()

Reply:
Ho god it's so crappy to have methods being both `api.model` & `api.multi`

Reply:
n'importe quoi

Reply:
1) ce n'est pas n'importe quoi. `api.model` = méthode statique
2) `_get_fields_onchange_subtotal` pour une méthode qui est appelée hors des onchanges, je ne suis pas d'accord non plus